augly/image/transforms.py

```diff
@@ -188,6 +188,52 @@ def apply_transform(
         return F.apply_pil_filter(image, filter_type=self.filter_type, metadata=metadata)
 
 
+class BarrelDistortion(BaseTransform):
```

Comment:
```suggestion
class DistortBarrel(BaseTransform):
```

nit: Generally we name augmentations the exact same name in functional & transforms. I would prefer we keep it that way to keep things crystal clear for users.